packages/integrations/netlify/CHANGELOG.md

```diff
@@ -34,7 +47,6 @@
   The new `Astro.clientAddress` property allows you to get the IP address of the requested user.
 
   ```astro
-  <div>Your address {Astro.clientAddress}</div>
```

Comment:
What happened here?

Reply:
Uh oh, I don't know! Prettier?

Reply:
Looks like the same bug we hit here https://github.com/withastro/astro/pull/4159/files#diff-e3d0a49bea0efda551d3bd7e5e860ab65409efbb9a419d274b5806ccacdd4061

Reply:
Manually fixed these for now. We'll have to investigate more!